front/src/Stores/ConsoleGlobalMessageManagerStore.ts

```diff
@@ -0,0 +1,5 @@
+import { writable } from "svelte/store";
+
+export const ConsoleGlobalMessageManagerVisibleStore = writable(false);
```

Comment:
Can we put this in camelCase (starting with a lower case character?) Because it is a variable name. => `consoleGlobalMessageManagerVisibleStore`